src/sentry/interfaces/contexts.py

```diff
@@ -28,8 +28,48 @@ def contexttype(cls):
 
 
 class ContextType:
-    indexed_fields = None
+    tag_promotion_mapping = None
```

Comment:
Should we be more explicit here and say `context_to_tag_mapping`? Promotion in this context is usually used to refer to promoting a tag/context to a column

Reply:
👍